src/strands/agent/agent_result.py

```diff
@@ -20,12 +22,14 @@ class AgentResult:
         message: The last message generated by the agent.
         metrics: Performance metrics collected during processing.
         state: Additional state information from the event loop.
+        structured_output: Parsed structured output when structured_output_type was specified.
     """
 
     stop_reason: StopReason
     message: Message
     metrics: EventLoopMetrics
     state: Any
+    structured_output: Optional[BaseModel] = None
```

Comment:
Thinking towards the future - is there any need for this to be a BaseModel?  Could this just be `Any | None`?